docs/malware-scanner/README.md

```diff
@@ -1 +1,44 @@
-## *Currently empty, check back later or contribute yourself*
\ No newline at end of file
+# Malware scanner and detector
+
+---
+
+# Dynamic analysis 
+A slow but reliable detector looking for jar capabilities, potential malware sign
+- run static analysis to filter known threats
+- downloading stuff from the internet (potentially jars, but can be genuine)
+- zip/jar manipulation.
+- able to execute stuff (Runtime.exec())
+- de-obfuscation, static string rebuilding from assets, byte arrays or StringBuilders
```

Comment:
2 cents: Deobufuscation isn't technically a necessary component with a tool like SSVM. With that, you can pattern match behaviors and ignore obfuscation entirely.